packages/react/src/primitives/Icon/internal.ts

```diff
@@ -1,2 +1,6 @@
 export * from './icons';
 export { useIcons } from './context/useIcons';
+export {
+  STORAGE_BROWSER_ICON_PATHS,
+  StorageBrowserIconType,
+} from './context/StorageBrowserIcons';
```

Comment:
Think this line is creating the public export

Reply:
Added export at suggested file